### ISSUE TITLE
xSQLServerAlwaysOnAvailabilityGroup: Error when changing FailoverMode from 'Manual' to 'Automatic'

### ISSUE THREAD

**Issue by cwegener** [PowerShell/xSQLServer#652]
**Details of the scenario you tried and the problem that is occurring:**

- When using xSQLServerAlwaysOnAvailabilityGroup to change the FailoverMode from Manual to Automatic, the following error message occurs:
```
Exception setting "AvailabilityMode": "Cannot convert value "Automatic" to type "Microsoft.SqlServer.Management.Smo.AvailabilityReplicaAvailabilityMode". Error: "Requested value 'automatic' was not found.""
```

**The DSC configuration that is using the resource (as detailed as possible):**
```
    availabilitymode= "SynchronousCommit"
    ensure= "present"
    failovermode= "automatic"
    name= "TestAG"
    sqlinstancename= "MSSQLSERVER"
    sqlserver= "LAB3SQL1"
```

**Version of the Operating System, SQL Server and PowerShell the DSC Target Node is running:**

- Microsoft Windows Server 2016 Standard Evaluation, version 10.0.14393.0

- Microsoft SQL Server 13.0.2204.0, RTM, Enterprise Evaluation Edition (64-bit)

- (Get-Host).Version: 5.1.14393.0

**What module (SqlServer or SQLPS) and which version of the module the DSC Target Node is running:**

- SQLPS (Version 1.0 ?).

**Version of the DSC module you're using, or 'dev' if you're using current dev branch:**

- 7.1.0.0

**Comment by cwegener** [comment]
PR #650 has been created to fix this.

**Comment by johlju** [comment]
This was merged in Pr #650.
